tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/model/NewLiferayPluginProjectOp.java

```diff
@@ -17,28 +17,28 @@
 import com.liferay.ide.core.ILiferayProjectProvider;
```

Comment:
I renamed them just  want to easily distinguish the class with name and I also want to reuse these codes what we have wrote service classes, then we just need to add a few different implementation of plugin project and module project and we don't need to maintain two copies of similar fucntion classes